src/report/report.py

```diff
@@ -571,6 +571,7 @@ def _get_issue_labels(
         job_name: Optional[str],
         type: str,
         jira_additional_labels: Optional[list[str]],
+        jira_additional_labels_filepath: str = f"{os.getenv('SHARED_DIR', '')}/firewatch-additional-labels",
```

Comment:
since it's  path please use `os.path.join`

Reply:
Ended up taking the path from a cli argument